docs/content/using/using_hostdefinition.md

```diff
@@ -0,0 +1,72 @@
+# Using dynamic host definition
+
+Dynamic host connectivity removes the necessity for manual host definitions. This is also facilitates the scaling out process of a node. 
```

Comment:
```suggestion
Dynamic host connectivity removes the necessity for manual host definitions. This also facilitates the scaling out process of a cluster. 
```

Reply:
and what do you think about:
`removes the necessity` -> `eliminates the onus`

Reply:
typo fixed